django/db/migrations/questioner.py

```diff
@@ -214,6 +219,30 @@ def ask_not_null_alteration(self, field_name, model_name):
                 return self._ask_default()
         return None
 
+    def ask_not_null_removal(self, field_name, model_name):
+        """Removing a NOT NULL field from a model"""
+        if not self.dry_run:
+            choice = self._choice_input(
+                f"It is not recommended to remove a non-nullable field '{field_name}' "
+                f"from '{model_name}' without providing a default."
+                f"This is because the database needs something to populate existing "
+                f"rows when migrating backwards.\n"
```

Comment:
Should we add an explicit mention of what would happen when migrating a `RemoveField` backwards? e.g. "When migrating backwards, the column/field will be re-added to the table."